docs/rhoas/rhoas-helm-guide/README.adoc

```diff
@@ -0,0 +1,395 @@
+
+.Prerequisites
+ifndef::community[]
+* You have a Red Hat account.
+endif::[]
+* You have a running Kafka instance in {product-kafka}.
+* You've installed the latest version of the `rhoas` CLI. See {base-url}{installation-guide-url-cli}[Installing and configuring the rhoas CLI^].
+* You've installed https://helm.sh/docs/intro/quickstart/:[Helm CLI^] version 3.9.0 or above
+* You've installed https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html:[OpenShift CLI] version 4.8.5 or above
+
+[id="proc-generating-configmap-for-application-services_{context}"]
+== Generate ConfigMap for Application Service Instances
+
+The application deployed in the Kubernetes platform needs to connect to the Kafka instance. To achieve this, one needs to supply the URL to the Kafka Instance. The URLs being non-confidential data can be supplied as Kubernetes ConfigMap. The helm chart can refer to the ConfigMap deployed in the OpenShift cluster which will henceforth connect the application to the Kafka Instance.
+
+This is implemented by RHOAS CLI using service contexts. In OpenShift Application Services, a service context is a defined set of instances running in cloud services such as OpenShift Streams for Apache Kafka and OpenShift Service Registry. You might create different contexts for specific use cases, projects, or environments. To learn more about contexts, you can go through this https://github.com/redhat-developer/app-services-guides/tree/main/docs/rhoas/rhoas-service-contexts:[guide].
+
+.Procedure
+
+. Confirm that the Managed Service instances are set in the current context and are running.
+
++
+[source,shell]
+----
+$ rhoas context status
+----
+
+. Generate a yaml file that contains the connection configuration as a Kubernetes ConfigMap object.
++
+[source,shell]
+----
+$ rhoas generate-config --type configmap --output-file ./rhoas-services.yaml
+----
++
+. Apply the generated ConfigMap file.
++
+[source,shell]
+----
+$ oc apply -f “./rhoas-services.yaml”
+----
++
+. Consuming ConfigMap in a Kubernetes deployment.
++
+An application deployed as a Kubernetes deployment can have the ConfigMap values injected as environment variables.
++
+[source,shell]
+----
+env:
+  - name: KAFKA_HOST
+    valueFrom: 
+      configMapKeyRef:
+        name: my-context-3-configuration
+        key: kafka_host
+  - name: SERVICE_REGISTRY_URL
+    valueFrom: 
+      configMapKeyRef:
+        name: my-context-3-configuration
+        key: service_registry_url
+----
+. Consuming ConfigMap in a Helm Chart.
++
+Helm charts support templating which implies ConfigMap values can be passed into templates. This will allow developers to quickly deploy their applications to OpenShift clusters. All one will need is - the Helm chart, Kubernetes ConfigMap and Kubernetes Secret. The helm chart needs to follow the naming standards set by RHOAS CLI. 
```

Comment:
Suggest:

"Helm charts support templating, which means that you can pass the ConfigMap values into 
a template. This enables you to quickly deploy applications to Kubernetes clusters. To do this, you need a Helm chart, a Kubernetes ConfigMap and a Kubernetes secret. The Helm chart needs to follow the naming standards set by the RHOAS CLI.